services/net/contentmigration/appsettings.json

```diff
@@ -93,6 +93,8 @@
         "The Northern View": "NV",
         "The Tri-City News": "TCN",
         "Victoria Times-Colonist": "TC",
+        "Victoria Times Colonist": "TC",
+        "Times Colonist": "TC",
```

Comment:
I hope this will resolve the issue where content from TNO is not getting put into the correct source.